src/java/org/apache/cassandra/repair/RepairSession.java

```diff
@@ -280,7 +276,7 @@ public void start(ListeningExecutorService executor)
         // Checking all nodes are live
         for (InetAddressAndPort endpoint : commonRange.endpoints)
         {
-            if (!FailureDetector.instance.isAlive(endpoint) && !skippedReplicas)
+            if (!FailureDetector.instance.isAlive(endpoint) && !commonRange.hasSkippedReplicas)
```

Comment:
This is nice. It makes it read, "The endpoint is alive and the common range has skipped replicas" without needing any additional context.